build

```diff
@@ -113,7 +122,7 @@ for PKG_PATH in $(catkin_topological_order --only-folders || colcon list --topol
   echo 11 > debian/compat
 
   # dpkg-source-opts: no need for upstream.tar.gz
-  sbuild --chroot-mode=unshare --no-clean-source --no-run-lintian \
+  sbuild --chroot-mode=unshare --no-clean-source --no-run-lintian --arch="$DEB_ARCH" \
```

Comment:
```suggestion
  sbuild --chroot-mode=unshare --no-clean-source --no-run-lintian \
```

Reply:
I don't see how this works. If we do not provide the `--arch` parameter, how does `sbuild` know which architecture to use?

Reply:
It uses the host architecture by default, just as the current `main` branch does.

Reply:
Ah, but we don't support the `$DEB_ARCH` from the environment then, new version coming.

Reply:
Add this before sbuild:
```shell
test -n "$DEB_ARCH" && set -- --arch="$OPTARG" "$@"
```

Reply:
this is still misisng.

Reply:
I squashed your suggestions into one commit and pushed it. Can you check this again?

The documentation for the `set` builtin starts with:
> This builtin is so complicated that it deserves its own section. set allows you to change the values of shell options and set the positional parameters, or to display the names and values of shell variables.

So I will not try to fully understand how the `set` builtin works on a weekend :-)

My superficial understanding is that `set` sets the command line options for the next command that will follow?

Reply:
> I squashed your suggestions into one commit and pushed it. Can you check this again?

looking good.

> So I will not try to fully understand how the `set` builtin works on a weekend :-)
> 
> My superficial understanding is that `set` sets the command line options for the next command that will follow?

It modifies the `$@` variable which is used in the `sbuild` call, later.

Reply:
With the recent changes, I get on my CI (https://github.com/christianrauch/camera-ros-deb-builder/actions/runs/8858847981/job/24327962826):
```
  E: Chroot for distribution jammy, architecture  not found
  E: Error creating chroot
```

Reply:
Fixed with recent commits.